corehq/motech/dhis2/entities_helpers.py

```diff
@@ -292,6 +312,12 @@ def register_tracked_entity_instance(requests, case_trigger_info, case_config):
     if case_updates:
         save_case_updates(requests.domain_name, case_trigger_info.case_id, case_updates)
 
+    if len(get_errors(response.json())):
+        raise Dhis2Exception(_(
+            'Error(s) while registering DHIS2 tracked entity. '
+            'Errors from DHIS2 can be found in Remote API Logs: {url}'
+        ).format(url=absolute_reverse(MotechLogListView.urlname, args=[requests.domain_name])))
```

Comment:
Another nitpick: I'd vote to rather use f-strings :thinking: [Here's](https://realpython.com/python-f-strings/#:~:text=As%20of%20Python%203.6%2C%20f,start%20using%20f%2Dstrings%20today.) some reasons why python's people would argue for this

Reply:
(I would normally agree, but this string is translated, and `gettext()` does not work with f-strings.)